code/game/gamemodes/miniantags/abduction/gland.dm

```diff
@@ -280,13 +280,13 @@
 	..()
 	if(ishuman(owner))
 		owner.gene_stability += GENE_INSTABILITY_MODERATE // give them this gene for free
-		owner.dna.SetSEState(GLOB.shockimmunityblock, TRUE)
+		owner.dna.SetDNAState(GLOB.shockimmunityblock, DNA_SE, TRUE)
 		genemutcheck(owner, GLOB.shockimmunityblock,  null, MUTCHK_FORCED)
 
 /obj/item/organ/internal/heart/gland/electric/remove(mob/living/carbon/M, special = 0)
 	if(ishuman(owner))
 		owner.gene_stability -= GENE_INSTABILITY_MODERATE // but return it to normal once it's removed
-		owner.dna.SetSEState(GLOB.shockimmunityblock, FALSE)
+		owner.dna.SetDNAState(GLOB.shockimmunityblock, FALSE, DNA_SE)
```

Comment:
There are two different orderings here.
I think the second one is right?

Reply:
Yep. This was a bad find-and-replace. Looks like I did it correctly for 0/FALSE. Really appreciate this review, seriously. Thank you so much.